make_docs.nu

```diff
@@ -136,3 +144,51 @@ $"($example.description)
   )
   $doc
 }
+
+def generate-category-sidebar [unique_categories] {
+  let sidebar_path = (['.', '.vuepress', 'configs', "sidebar", "command_categories.ts"] | path join)
+  let list_content = (
+    $unique_categories |
+    each { |category| $category | str replace '\?' '' | str replace ' ' '_' } |
+    each { |category| $"  '/commands/categories/($category).md',"} |
+    str join (char newline)
+  )
+  $"export const commandCategories = [
+($list_content)
+];
+" | save --raw --force $sidebar_path
+}
+
+
+def generate-category [category] {
+  let safe_name = ($category | str replace '\?' '' | str replace ' ' '_')
+  let doc_path = (['.', 'commands', 'categories', $'($safe_name).md'] | path join)
+  $"# ($category)
```

Comment:
`($category | str title-case)` will make docs look better

Reply:
Nice, I was looking for something like this, but missed it.